resources/css/app.css

```diff
@@ -0,0 +1,7 @@
+@import "material-icons";
+@import "bootstrap";
```

Comment:
Will you try to get rid of bootstrap? From what I see at least the login page is already built, I don't think we need extra styling, so I don't see the need for bootstrap as of now :thinking: